packages/ripple-binary-codec/test/binary-parser.test.ts

```diff
@@ -238,19 +238,19 @@ function fieldParsingTests() {
   it('Field throws when type code out of range', () => {
     const parser = makeParser('0101')
     expect(() => parser.readField()).toThrow(
-      new Error('Cannot read FieldOrdinal, type_code out of range'),
+      new Error('Cannot read FieldOrdinal, type_code 1 out of range'),
     )
   })
   it('Field throws when field code out of range', () => {
     const parser = makeParser('1001')
     expect(() => parser.readFieldOrdinal()).toThrow(
-      new Error('Cannot read FieldOrdinal, field_code out of range'),
+      new Error('Cannot read FieldOrdinal, field_code 1 out of range'),
     )
   })
   it('Field throws when both type and field code out of range', () => {
     const parser = makeParser('000101')
     expect(() => parser.readFieldOrdinal()).toThrow(
-      new Error('Cannot read FieldOrdinal, type_code out of range'),
+      new Error('Cannot read FieldOrdinal, type_code 1 out of range'),
```

Comment:
`1 out of range` doesn't look correct.. I also have no idea how this works.

Reply:
in my local runs of this test, I observe the value `1`, as reported in these print statements. But I don't understand the parsing logic.

Reply:
It doesn't matter, the only reason these test changes are needed are due to the better error reporting in `binary-parser.ts`. This test is written to cause those errors.